roles/tests/test_permissions.py

```diff
@@ -57,17 +59,27 @@ def test_repo_permissions(self):
             RepoPermission.add_edit_metadata,
             add_edit_metadata
         )
+        self.assertEqual(
+            RepoPermission.manage_repo_users,
+            manage_repo_users
+        )
         self.assertListEqual(
             RepoPermission.administrator_permissions(),
-            [view_repo[0], import_course[0],
-             manage_taxonomy[0], add_edit_metadata[0]]
+            [view_repo[0],
+             import_course[0],
+             manage_taxonomy[0],
+             add_edit_metadata[0],
+             manage_repo_users[0]]
```

Comment:
nit: still using `[<item>`, instead of
```
[
    <item>,
```
so it isn't quite as easy to add a new item...again, total nit.